benchmarks/bm_runner.py

```diff
@@ -562,11 +619,24 @@ def add_asv_arguments(self) -> None:
 def main():
     parser = ArgumentParser(
         description="Run the Iris performance benchmarks (using Airspeed Velocity).",
-        epilog="More help is available within each sub-command.",
+        epilog=(
+            "More help is available within each sub-command."
+            "\n\nNOTE(1): iris-test-data is downloaded and cached within the "
+            "benchmarks code directory.\n   Set $OVERRIDE_TEST_DATA_REPOSITORY "
```

Comment:
It's actually in the `DATA_GEN_PYTHON` environment, so this message needs tweaking and would make more sense to come after the current `NOTE(2)`.